uaclient/conftest.py

```diff
@@ -33,12 +33,14 @@ def caplog_text(request):
             # Older versions of pytest only have getfuncargvalue, which is now
             # deprecated in favour of getfixturevalue
             caplog = request.getfuncargvalue('caplog')
+        caplog.set_level(logging.INFO)
```

Comment:
I think the changes to this file fix #340?

(For reference, in future, ideally I'd like to see changes like this in a separate commit. It makes it easier to understand what the motivation for a change was if it has a more specific commit message, and it also means that we won't regress #340 if we ever wanted to revert the commit that introduces these changes. To be clear, I *don't* think it's worth splitting it in to a separate commit in this PR.)